gaia-dialog-alert.js

```diff
@@ -17,6 +17,8 @@ module.exports = component.register('gaia-dialog-alert', {
     this.els = {
       dialog: this.shadowRoot.querySelector('gaia-dialog')
     };
+    this.els.dialog.addEventListener('closed',
+                                     GaiaDialogProto.hide.bind(this));
```

Comment:
What is this line actually for? Why do we have to call `.hide()` again after the close animation has finished?